modules/hcp-configure-ntp.adoc

```diff
@@ -0,0 +1,170 @@
+====
+
+. Use Butane to generate a `MachineConfig` object file, `99-worker-chrony.yaml`, that contains a configuration to be sent to the nodes. Run the following command:
++
+[source,terminal]
+----
+$ butane 99-worker-chrony.bu -o 99-worker-chrony.yaml
+----
++
+.Example `99-worker-chrony.yaml` configuration
+[source,yaml]
+----
+# Generated by Butane; do not edit
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: <machineconfig_name>
+spec:
+  config:
+    ignition:
+      version: 3.2.0
+    storage:
+      files:
+        - contents:
+            source: data:...
+          mode: 420
+          overwrite: true
+          path: /example/path
+----
+
+. Add the contents of the `99-worker-chrony.yaml` file inside of a config map in the management cluster:
++
+.Example config map
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: <configmap_name>
+  namespace: <namespace>  #<1>
+data:
+  config: |
+    apiVersion: machineconfiguration.openshift.io/v1
+    kind: MachineConfig
+    metadata:
+      labels:
+        machineconfiguration.openshift.io/role: worker
+      name: <machineconfig_name>
+    spec:
+      config:
+        ignition:
+          version: 3.2.0
+        storage:
+          files:
+          - contents:
+              source: data:...
+            mode: 420
+            overwrite: true
+            path: /example/path
+----
+<1> Replace `<namespace>` with the name of your namespace where you created the node pool, for example, `clusters`.
+
+. Apply the config map to your node pool by running the following command:
++
+[source,terminal]
+----
+$ oc edit nodepool <nodepool_name> --namespace <hosted_cluster_namespace>
+----
++
+.Example `NodePool` configuration
+[source,yaml]
+----
+apiVersion: hypershift.openshift.io/v1alpha1
+kind: NodePool
+metadata:
+# ...
+  name: nodepool-1
+  namespace: clusters
+# ...
+spec:
+  config:
+  - name: <configmap_name> #<1>
+# ...
+----
+<1> Replace `<configmap_name>` with the name of your config map.
+
+. Add the list of your NTP servers in the `infra-env.yaml` file, that defines the `InfraEnv` custom resource (CR):
++
+.Example `infra-env.yaml` file
+[source,yaml]
+----
+apiVersion: agent-install.openshift.io/v1beta1
+kind: InfraEnv
+# ...
+spec:
+  additionalNTPSources:
+  - <ntp_server> #<1>
+  - <ntp_server1>
+  - <ntp_server2>
+# ...
+----
+<1> Replace `<ntp_server>` with the name of your NTP server. For more details about creating a host inventory and the `InfraEnv` CR, see "Creating a host inventory".
+
+. Apply the `InfraEnv` CR by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f infra-env.yaml
+----
+
+.Verification
+
+* Verify that your host inventory is created by running the following command. Check the following fields in the command output:
```

Comment:
I do not see a command here but instead descriptions of output parameters.

Reply:
Remove "Verify that your host inventory is created by running the following command"?

Reply:
The command is placed after parameter descriptions.  `$ oc describe infraenv <infraenv_resource_name> -n <infraenv_namespace>`  
I will shift `Verify that your....by running the...` above the command instead of mentioning it in the beginning.